pkg/engine/kube.go

```diff
@@ -23,81 +24,158 @@ type YamlMeta struct {
 	Kind       string `yaml:"kind"`
 }
 
-func kubePodman(ctx context.Context, path string) error {
-	klog.Infof("Creating podman container from %s using kube method", path)
-
-	kubeYaml, err := ioutil.ReadFile(path)
+func kubePodman(ctx context.Context, path *string, prev *string) error {
+	if path != nil {
+		klog.Infof("Creating podman container from %s using kube method", *path)
+	}
+	ctx, err := bindings.NewConnection(ctx, "unix://run/podman/podman.sock")
```

Comment:
also, we have to keep in mind that if the config specifies a `directory` for TargetPath, like `examples/kube` then, if the only file in that directory is removed, upon a git push, the directory is not pushed and so no longer exists. So, when I remove example.yaml, which is the only file in the directory, I see the `no such file or directory` when looking for `examples/kube/example.yaml` but that is bc `examples/kube` no longer exists, rather than `examples/kube/example.yaml` no longer exists. I think we'll have to handle that differently?